hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemTransactions.java

```diff
@@ -386,17 +385,14 @@ ctx, createFileID(filesConfig.hapiPermissions(), config), bytes),
             final var node = nodeStore.get(i);
             final var nodeInfo = networkInfo.nodeInfo(i);
             if (nodeInfo != null && node != null && !node.deleted()) {
-                final var declineReward = nodeInfo.accountId().accountNumOrThrow() <= ledgerConfig.numSystemAccounts();
-                if (declineReward) {
-                    log.info(
-                            "Updating node{} with system node account {} to decline rewards",
-                            nodeInfo.nodeId(),
-                            nodeInfo.accountId());
-                    systemContext.dispatchAdmin(b -> b.nodeUpdate(NodeUpdateTransactionBody.newBuilder()
-                            .nodeId(nodeInfo.nodeId())
-                            .declineReward(true)
-                            .build()));
-                }
+                log.info(
```

Comment:
```
// (FUTURE) Remove this 0.61-specific code initiating all system node accounts to decline rewards
```

I just noticed there's a comment further up ☝️ about removing 'this' code. Not sure what the context is, does this comment have any relevance here? Should this `if` block just be removed?

Reply:
Yeah it doesnt hurt to leave it here. But should definitely be removed after 0.62